packages/apps-config/src/endpoints/productionRelayKusama.ts

```diff
@@ -48,19 +48,19 @@ export function createKusama (t: TFunction): EndpointOption {
         }
       },
       {
-        info: 'crab',
-        paraId: 2006,
-        text: t('rpc.kusama.crab', 'Darwinia Crab', { ns: 'apps-config' }),
+        info: 'shadow',
```

Comment:
> Added alphabetical based on chain name
I added with Chain name started with `C`, is that ok with different info and chain name?

Reply:
All fine. The "info" just maps through to knowing which logo to use.